Libraries/Hotcakes.Commerce.Dnn/Web/HotcakesModuleBase.cs

```diff
@@ -160,6 +160,12 @@ protected void RegisterScript(string path, int order = 0)
                 FileOrder.Js.DefaultPriority + order);
         }
 
+        protected void RegisterExternalScript(string path, int order = 0)
```

Comment:
Please rename this to `RegisterScriptExternal` to better match the existing naming convention.